From 3a3dff3507830e2fd17c53c65aad394e40a83af2 Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Sun, 4 Oct 2020 00:02:24 -0400
Subject: [PATCH] Move pytest-runner to tests-require

---
 setup.py | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/setup.py b/setup.py
index 83a2052..ea960f1 100644
--- a/setup.py
+++ b/setup.py
@@ -31,8 +31,7 @@ if use_cython:
 
 setup(
     ext_modules = extensions,
-    setup_requires=["pytest-runner"],
-    tests_require=["pytest", "hypothesis"],
+    tests_require=["pytest", "hypothesis", "pytest-runner"],
     name="pytiff",
     version=_version.__version__,
     packages=["pytiff", "pytiff._version"],
-- 
2.26.2

